db/warehouse/migrate/20250121204933_add_hmis_household_events_table.rb

```diff
@@ -0,0 +1,22 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+class AddHmisHouseholdEventsTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :hmis_household_events do |t|
+      t.string :HouseholdID, null: false
+      t.references :data_source, null: false
+      t.string :event_type, null: false
+      t.jsonb :event_details
+      t.references :user, null: false
+      t.timestamps
+    end
```

Comment:
Tagging @eanders as an FYI on this new HMIS table which tracks changes to households.

For now, the events are "join" and "merge", with the idea that we may also want to track "add" "remove" "hoh change" etc.

In the future, this could back a neat "household history" interface.

Reply:
Thanks for the tag!  That sounds like a wonderful plan.